examples/export/export.cy.js

```diff
@@ -0,0 +1,29 @@
+describe('Tram-Lite Example Components (via export)', () => {
```

Comment:
Along with making the new example/test pages, we've also moved the cypress tests to be adjacent to the page files. This mimics the pattern common with unit-tests (where the tests are adjacent to the source code).